docs/usage.md

```diff
@@ -11,7 +11,7 @@
   * [Reproducibility](#reproducibility)
 * [Main arguments](#main-arguments)
   * [`-profile`](#-profile)
-  * [metabolomics specific parameters (**how to run the workflow**)](#parameters-file)
+  * [metabolomics specific parameters (**how to run the workflow**)](#how-to-run-the-workflow)
```

Comment:
Maybe worth setting a link to the centralized docs pages:
https://nf-co.re/metaboigniter/usage

I thought the idea was to move all documentation to this centralized docs page on the long run.

Reply:
Great comment. Where would you suggest placing the link? Somewhere in the document or already in the beginning?

Reply:
Thanks for the comments by the way

Reply:
No worries, I guess its partially up to you - as far as I know most pipelines include the link even above the table of contents for example like this:

Please read this documentation on the nf-core website: [https://nf-co.re/metaboigniter/usage](https://nf-co.re/metaboigniter/usage)